ardesp/waterelf32/AdafruitIFTTT.h

```diff
@@ -0,0 +1,51 @@
+// Ex20.h
+
+#ifndef AdafruitIFTTT
+#define AdafruitIFTTT
+
+#define IO_USERNAME    "andrisole"
+#define IO_KEY         "63cc3971e9a044c0b7c54dae5a2ffaff"
+
+/******************************* WIFI **************************************/
+
+// the AdafruitIO_WiFi client will work with the following boards:
+//   - HUZZAH ESP8266 Breakout -> https://www.adafruit.com/products/2471
+//   - Feather HUZZAH ESP8266 -> https://www.adafruit.com/products/2821
+//   - Feather M0 WiFi -> https://www.adafruit.com/products/3010
+//   - Feather WICED -> https://www.adafruit.com/products/3056
+
+#define WIFI_SSID       "<WIFI_SSID>"
+#define WIFI_PASS       "<WIFI_PASS>"
+#define IFTTT_KEY       "<IFTTT_KEY>"
+#define IFTTT_EVENT_NAME       "<IFTTT_EVENT_NAME>"
+#include "AdafruitIO_WiFi.h"
+#include <WiFi.h>
+#include <AdafruitIO_Feed.h>
+#include <ArduinoHttpClient.h>
+// Connecting to Adafruit.IO APIs
+AdafruitIO_WiFi io(IO_USERNAME, IO_KEY, WIFI_SSID, WIFI_PASS);
+WiFiClient wifi;
+//Adafruit and IFTTT clients
+HttpClient ioClient = HttpClient(wifi, "io.adafruit.com", 80);
```

Comment:
Do `ioClient` and `IFTTTClient` need to be accessible to consumers of this library/file? If not ideally they would be declared and initialised in the `.cpp` file (and also declared `static` to avoid global variable name collisions).

Reply:
Hi @Eroc33 ,
In theory, they can be accessible, I don't see any conflicts with them. There should be however only 1 instance of each.

I did not focus on object-oriented design for this assignment, and I am first time working with C, but thanks for the feedback! 👍